src/probabilistic.jl

```diff
@@ -544,3 +544,85 @@ $DOC_DISTRIBUTIONS
+        new(nbins, bin_overlap, min, max, cor)
+    end
+end
+
+ContinuousBoyceIndex(; kw...) = _ContinuousBoyceIndex(; kw...) |> robust_measure |> fussy_measure
+
+function (m::_ContinuousBoyceIndex)(ŷ::UnivariateFiniteArray, y::NonMissingCatArrOrSub; warn=true)
+    warn && warn_unordered(levels(y))
+    positive_class = levels(first(ŷ))|> last
+    scores = pdf.(ŷ, positive_class)
+    max = isnothing(m.max) ? maximum(scores) : m.max
+    min = isnothing(m.min) ? minimum(scores) : m.min
+    binwidth = m.bin_overlap * (max - min)
+
+    return Functions.cbi(scores, y, positive_class; nbins = m.nbins, binwidth, max, min, cor = m.cor)
+end
+
+const ContinuousBoyceIndexType = API.FussyMeasure{<:API.RobustMeasure{<:_ContinuousBoyceIndex}}
+
+@fix_show ContinuousBoyceIndex::ContinuousBoyceIndexType
+
+StatisticalMeasures.@trait(
+    _ContinuousBoyceIndex,
+    consumes_multiple_observations=true,
+    observation_scitype = Finite{2},
+    kind_of_proxy=StatisticalMeasures.LearnAPI.Distribution(),
+    orientation=Score(),
+    external_aggregation_mode=Mean(),
+    human_name = "continuous boyce index",
+)
+
+register(ContinuousBoyceIndex, "continuous_boyce_index", "cbi")
+
+const ContinuousBoyceIndexDoc = docstring(
+    "ContinuousBoyceIndex(; nbins=101, bin_overlap=0.1, min=nothing, max=nothing, cor=StatsBase.corspearman)",
+    body=
+"""
+The Continuous Boyce Index is a measure for evaluating the performance of probabilistic predictions for binary classification, 
+especially for presence-background data in ecological modeling. 
+It compares the predicted probability scores for the positive class across bins, giving higher scores if the ratio of positive
+    and negative samples in each bin is strongly correlated to the value at that bin.
+
+## Keywords
+
+- `nbins`: Number of bins to use for score partitioning.
+- `bin_overlap`: Relative overlap between bins. The actual bin width is `bin_overlap * (max - min)`.
+- `min`, `max`: Optional minimum and maximum score values for binning. Default to the
+    maximum and minimum values of the scores.
+- `cor`: Correlation function (defaults to StatsBase.corspearman, i.e. Spearman correlation).
+
+## Arguments
+
+The predictions `ŷ` should be a vector of `UnivariateFinite` distributions from CategoricalDistributions.jl, and `y` a vector of ground truth labels.
+
+Returns the correlation between the ratio of positive to negative samples in each bin and the bin centers.
+
+Core implementation: [`Functions.cbi`](@ref).
+
+Reference:
+Alexandre H. Hirzel, Gwenaëlle Le Lay, Véronique Helfer, Christophe Randin, Antoine Guisan,
+Evaluating the ability of habitat suitability models to predict species presences,
+Ecological Modelling,
+Volume 199, Issue 2, 2006
+""",
+    scitype="",
+)
+
+"$ContinuousBoyceIndexDoc"
+ContinuousBoyceIndex
+"$ContinuousBoyceIndexDoc"
+const cbi(x, y) = ContinuousBoyceIndex()(x, y)
```

Comment:
The alias should be an *instance* of the type. And I think you forgot `continuous_boyce_index`:

```suggestion
"$ContinuousBoyceIndexDoc"
const cbi = ContinuousBoyceIndex()
"$ContinuousBoyceIndexDoc"
const continuous_boyce_index = ContinuousBoyceIndex()
```